src/test/java/seedu/duke/task/TaskListTest.java

```diff
@@ -57,7 +57,7 @@ public void deleteTask_negativeIndex_exceptionThrown() {
 
     @Test
     public void markTaskAsDone_indexOutOfBounds_exceptionThrown() {
-        assertThrows(DukeException.class, () -> {
+        assertThrows(IndexOutOfBoundsException.class, () -> {

```

Comment:
Generally, I think we should mask everything (except `IOException`) to `DukeException` for the consistency of our program.